website_customer_also_purchased/views/layout.xml

```diff
@@ -0,0 +1,12 @@
+<openerp>
+    <data>
+		<template id="assets_frontend" inherit_id="website.assets_frontend" name="Shop Stock State">
+  			<xpath expr="." position="inside">
+                <link rel='stylesheet' href='/website_customer_also_purchased/static/src/js/node_modules/slick-carousel/slick/slick.less'/>
```

Comment:
http://kenwheeler.github.io/slick/
slick: have more features than thumbnail. Dragging, responsive design and  carousel methods implemented.